netbout-web/src/main/java/com/netbout/dynamo/DyAlias.java

```diff
@@ -70,15 +81,6 @@
      */
     private final transient Item item;
 
-    static {
```

Comment:
@mkordas Se this discussion https://github.com/yegor256/netbout/pull/1017#discussion-diff-53019201